### DISCUSSION TITLE
Questions about jax-101 tutorials 05 random number generation

### DISCUSSION THREAD

**Discussion by JiahaoYao** [jax-ml/jax#6156] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/20907377/111891020-55b15f80-89ac-11eb-89c5-2579a919e8b5.png)

First, thanks for making the wonderful tutorials. i have a silly question about this statement. Here, it says that the `numpy` state is updated by each call to a random function, but why here the second call does not change the state? Thanks if someone can let me know.

**Comment by jakevdp** [comment, marked as answer]
The state is changed, but the function only prints the first 400 or so characters of the string representation of the state. If you print the full state, you can see where the changes are.

**Reply by JiahaoYao** [reply]
Thanks @jakevdp , got that!
